### ISSUE TITLE
"unexpected token" on startup

### ISSUE THREAD

**Issue by radix** [yakyak/yakyak#732]
I can't start YakYak any more. I get a dialog with the following text:

"A JavaScript error occurred in the main process"

"Uncaught exception"
"SyntaxError: Unexpected token"

I've tried reinstalling the latest release and I'm still getting it.

**Comment by kukulich** [comment]
I have same problem.

**Comment by tesseract241** [comment]
Same problem here, it happened after the lights went out in my apartment. I even deleted the entirety of the yakyak folder (Windows 10 here), but the same error happens.

**Comment by kukulich** [comment]
I've removed all yakyak data in my user profile and now it works again :)

**Comment by tesseract241** [comment]
Thanks for mentioning it, it worked for me too.  
To help other, under windows the profile data is stored in AppData/Roaming/YakYak in your personal folder.
AppData itself is a hidden folder, which you need to tell Windows you want to be able to see by going under Folder Preferences.

**Comment by radix** [comment]
yes, deleting the folder does work around the problem.

I noticed that other people have filed a similar bug on the tracker, but for some reason they closed the bug again when they figured out how to reset their profile data. Of course, the bug is still there, so I'm not sure why they closed the bug...

**Comment by hmetin07** [comment]
Have you tried deleting %AppData%\yakyak folder ?

https://goo.gl/HE7ooD

**Comment by Guardian9978** [comment]
I had this problem to so I went into %appdata%/yakyak and I made a backup of all the files/folder and then tried to start it up with different combinations until i found the problem file "cookies.json" when I opened it with notepad++ it was full of "nul" entries. So for me the fix for this problem was to delete cookies.json from %appdata%/yakyak and the problem went away.

**Comment by uwu-uwu** [comment]
Same problem here, had to delete app data folder, during a sudden power interruption causing the pc to shut off.

**Comment by luni3359** [comment]
I delete the folder constantly, it's becoming an issue.

**Comment by averissimo** [comment]
I haven't deleted mine in ages, I haven't experience power problems, just memory ones that should be equivalent.

For those that this is a recurrent issue if that happens again could you not delete the folder, just change names so you can continue using it.

So that you can provide feedback directly with an invalid folder

**Comment by Viper007Bond** [comment]
Bad bot. Still a valid issue and could be worked around by discarding corrupted files.

**Comment by arcooke** [comment]
Deleting data folder fixes it for me as well.  On Linux, the location is `~/.local/share/applications/yakyak` .. just delete the folder and everything will re-generate and the app will start up again.

**Comment by jrkoch** [comment]
Thank you so much for providing the solution.  It's happening to me as well.

**Comment by averissimo** [comment]
@jrkoch We would greatly appreciate the contribution!!

**Comment by jakubsuchybio** [comment]
I happened to me for the third time. I think I just found a pattern when it happens. 
(Ofc I know the workaround via deleting user data) 
It happened to me this last time when electricity got cut of of my desktop PC.

**Comment by Viper007Bond** [comment]
Yeah, it only happens to me when I have to forcibly power off my laptop due to it freezing.

**Comment by robkorv** [comment]
I am running my own build on Ubuntu. 

`rm -Rf ~/.config/yakyak` fixed the problem for me.

Had to login again though, but that's better then not able to run it at all,

**Comment by Viper007Bond** [comment]
@robkorv You only need to delete the `cookies.json` file, not the whole folder.

**Comment by arcooke** [comment]
@radix 

> yes, deleting the folder does work around the problem.

> I noticed that other people have filed a similar bug on the tracker, but for some reason they closed the bug again when they figured out how to reset their profile data. Of course, the bug is still there, so I'm not sure why they closed the bug...

Speaking of which.. this stale bot isn't helping ;)

**Comment by Viper007Bond** [comment]
Bad bot.

**Comment by elvarb** [comment]
Same issue, deleting cookies.json in the appdata folder works
